src/sentry/static/sentry/app/components/events/eventTags.jsx

```diff
@@ -26,7 +27,18 @@ class EventTags extends React.Component {
     const tags = this.props.event.tags;
     if (_.isEmpty(tags)) return null;
 
-    const {orgId, projectId} = this.props;
+    const {organization, orgId, projectId} = this.props;
+
+    const hasSentry10 = new Set(organization.features).has('sentry10');
+
+    const streamPath = hasSentry10
+      ? `/organizations/${orgId}/issues/`
+      : `/${orgId}/${projectId}/`;
+
+    const releasesPath = hasSentry10
+      ? `/organizations/${orgId}/releases/`
+      : `/${orgId}/${projectId}/releases/`;
```

Comment:
Should these URLs include the event's project id in the querystring?

Reply:
Hmm, we could but I'd rather leave this as-is. I kinda see these as just shortcuts to populate issue queries, rather than changing the global selections